tools/dotnet-linker/ApplyPreserveAttributeBase.cs

```diff
@@ -143,12 +143,6 @@ void PreserveUnconditional (IMetadataTokenProvider provider)
 		{
 			Annotations.Mark (provider);
 			AddDynamicDependencyAttribute (provider);
-
-			var member = provider as IMemberDefinition;
-			if (member is null || member.DeclaringType is null)
-				return;
-
-			Mark (member.DeclaringType, null);
```

Comment:
This changes the original code to mark Preserved code, so I'm not sure I'm comfortable with that.

Could we instead just skip adding the attribute in the AddDynamicDependencyAttribute method below instead?

Reply:
I see, I will change the PR to only skip the DynamicDependency attribute and keep the original marking logic.